src/index.tsx

```diff
@@ -5,6 +5,11 @@ import '@fortawesome/fontawesome-free/css/all.css';
 import './styles/index.scss';
 
 import { App } from './App';
+import { AppTodoProvider } from './components/AppTodoContext/AppTodoContext';
```

Comment:
It's a good practice to keep context providers in a separate file, like `contexts.tsx`. Consider moving `AppTodoProvider` import to a `contexts.tsx` file and import it from there. 📁